emr-dynamodb-hadoop/src/test/java/org/apache/hadoop/dynamodb/DynamoDBClientTest.java

```diff
@@ -171,14 +171,14 @@ public void throwsWhenGivenProxyUsernameAndPasswordWithoutProxyHostAndPortAreNot
   public void testPutBatchThrowsWhenItemIsTooLarge() throws Exception {
     Map<String, AttributeValue> item = ImmutableMap.of("",
         new AttributeValue(Strings.repeat("a", (int) (DEFAULT_MAX_ITEM_SIZE + 1))));
-    client.putBatch("dummyTable", item, 1, null);
+    client.putBatch("dummyTable", item, 1, null, false);
```

Comment:
[suggestion] please come up test case(s) when deletion mode is true, and verify it's calling the `DeleteRequest()` (to help others and guard for this feature in the future won't be untracked.)

Reply:
Done!